src/Home.tsx

```diff
@@ -0,0 +1,21 @@
+import React,{useState} from 'react';
+import './style/home.scss';
+
+export function HomePage() {
+const[homeHeight, setHomeHeight] = useState( {height:"100vh"})
+
+function setHomeHeighttoZero(){
+    setHomeHeight({height:"100px"});
+}
+
+    return (
+        <div >
+            <section className="home" onClick={setHomeHeighttoZero} style={homeHeight}> Welcome to the Mars Rover!</section>
```

Comment:
How do people know that they should click this?
What happens if they are only using a keyboard?